indy_node/test/nym_txn/test_demote_network_monitor.py

```diff
@@ -0,0 +1,72 @@
+                                                       sdk_wallet_trustee,
+                                                       sdk_wallet_handle,
+                                                       with_verkey):
+    new_steward_did, new_steward_verkey = looper.loop.run_until_complete(
+        did.create_and_store_my_did(sdk_wallet_trustee[0], "{}"))
+    new_network_monitor_did, new_network_monitor_verkey = looper.loop.run_until_complete(
+        did.create_and_store_my_did(sdk_wallet_steward[0], "{}"))
+    """Adding new steward"""
+    sdk_add_new_nym(looper, sdk_pool_handle,
+                    sdk_wallet_trustee, 'newSteward', STEWARD_STRING, verkey=new_steward_verkey, dest=new_steward_did)
+    """Adding NETWORK_MONITOR role by first steward"""
+    op = {'type': '1',
+          'dest': new_network_monitor_did,
+          'role': NETWORK_MONITOR,
+          'verkey': new_network_monitor_verkey}
+    req = sdk_sign_and_submit_op(looper, sdk_pool_handle, (sdk_wallet_handle, new_steward_did), op)
+    sdk_get_and_check_replies(looper, [req])
+    """Check that get_validator_info command works for NETWORK_MONITOR role"""
+    sdk_get_validator_info(looper, (sdk_wallet_handle, new_network_monitor_did), sdk_pool_handle)
+    op = {'type': '1',
+          'dest': new_network_monitor_did,
+          'role': None}
+    if with_verkey:
+        op['verkey'] = new_network_monitor_verkey
+    """Blacklisting network_monitor by new steward"""
+    req = sdk_sign_and_submit_op(looper, sdk_pool_handle, (sdk_wallet_handle, new_steward_did), op)
+    if with_verkey:
+        with pytest.raises(RequestRejectedException):
+            sdk_get_and_check_replies(looper, [req])
+    else:
+        sdk_get_and_check_replies(looper, [req])
+        with pytest.raises(RequestRejectedException):
+            sdk_get_validator_info(looper, (sdk_wallet_handle, new_network_monitor_did), sdk_pool_handle)
+
+
+def test_network_monitor_suspension_by_itself(looper,
+                                              sdk_pool_handle,
+                                              sdk_wallet_steward,
+                                              sdk_wallet_handle,
+                                              with_verkey):
+    new_network_monitor_did, new_network_monitor_verkey = looper.loop.run_until_complete(
+        did.create_and_store_my_did(sdk_wallet_steward[0], "{}"))
+    """Adding NETWORK_MONITOR role by steward"""
+    op = {'type': '1',
+          'dest': new_network_monitor_did,
+          'role': NETWORK_MONITOR,
+          'verkey': new_network_monitor_verkey}
+    req = sdk_sign_and_submit_op(looper, sdk_pool_handle, (sdk_wallet_handle, sdk_wallet_steward[1]), op)
+    sdk_get_and_check_replies(looper, [req])
+    op = {'type': '1',
+          'dest': new_network_monitor_did,
+          'role': None}
+    if with_verkey:
+        op['verkey'] = new_network_monitor_verkey
+    """Blacklisting network_monitor by itself"""
```

Comment:
I might be wrong, but it seems like this comment belongs before op assignment, not after